Makefile

```diff
@@ -116,8 +116,8 @@ golangci-lint:
 	$(LOCALBIN)/golangci-lint run --fix --verbose
 
 .PHONY: test
-test: manifests generate fmt vet envtest ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./... -coverprofile cover.out
+test: manifests generate fmt vet envtest ginkgo ## Run tests.
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" $(GINKGO) --trace --cover --coverpkg=../../pkg/keystone,../../controllers,../../api/v1beta1 --coverprofile cover.out --covermode=atomic ${PROC_CMD} $(GINKGO_ARGS) ./tests/...
```

Comment:
If you add PROC_CMD the the ginkgo command line then I guess you also want to set it to something. In nova-operator we calculate it with:
```
PROCS?=$(shell expr $(shell nproc --ignore 2) / 2)
PROC_CMD = --procs ${PROCS}
```

Reply:
Ahh, I didn't notice these and was wondering why we need that PROC_CMD. Fixed the command.